regtests/bin/matrix.base

```diff
@@ -2010,18 +2010,18 @@
  if [ "$assim" = 'y' ]
  then
    echo ' ' >> matrix.body
-   echo "$rtst -s ST4 -w work_UPD0F_O    -i input_UPD0F_O    $ww3 ww3_ta1" >> matrix.body
+   echo "$rtst -s ST4 -N -w work_UPD0F_O    -i input_UPD0F_O    $ww3 ww3_ta1" >> matrix.body
```

Comment:
I wonder if rather than hardcoding -N here, we should allow the user to set it via the $rtst environment variable?

Reply:
I think there is benefits to limiting the user defined options for the regtests, may give more consistency and they are unlikely to want/need to change the .inp or .nml files for the regtest.

Reply:
I suppose my reasoning is that all the other regtests (bar a few) do not have the `-N` option. With it hardcoded here, the user running the regtest matrix user can't **un-set** it if they want to run ising `.inp` files

I know that Ifremer set the -N switch in their matrix_datamor file ($rtst). I feel that is a better place to set it, maybe? Saying that, I know there is a push to move all regtests to use NML options - perhaps @mickaelaccensi can comment?

Reply:
I will push a branch to use namelist as default for all regtests, so it will be the default behavior for regtest runs in the coming weeks. For this PR, -N option can be set for this specific regtest. Once namelist development is done for all ww3 programs, the idea is to remove inp from regtests.

Reply:
In that case it's all good :)
Thanks @mickaelaccensi .

Reply:
Thanks @mickaelaccensi . So perhaps I'll leave the hardcoded -N in for the time being, and this can be updated/removed as part of the subsequent branch development. The other option would be to remove and allow all regtests to use .inp for the time being; I've tested enough to be happy that the .nml files will work at a later date and they are available for all the ww3_ta1 regtests in your subsequent branch

Reply:
Thanks @ukmo-ccbunney will leave as currently coded then!